Dockerfile.apache

```diff
@@ -48,6 +48,13 @@ RUN docker-php-ext-install pdo_mysql pdo_pgsql pdo_sqlite
 RUN docker-php-ext-install soap
 RUN docker-php-ext-install tokenizer
 RUN docker-php-ext-install zip
+RUN apt-get install -y zlib1g-dev libicu-dev g++
```

Comment:
Move this line up above the php config section and into UPDATE/UPGRADE APT PACKAGES section  ( round line: 34